source/read/change-streams.txt

```diff
@@ -0,0 +1,254 @@
+
+.. literalinclude:: /includes/read/change-streams.php
+   :start-after: start-update-for-change-stream
+   :end-before: end-update-for-change-stream
+   :language: php
+   :dedent:
+
+When you update the collection, the change stream application prints the change
+as it occurs. The printed change event resembles the following output:
+
+.. code-block:: bash
+   :copyable: false
+
+   {"_id":{"_data":"..."},"operationType":"update","clusterTime":{"$timestamp":{...},
+   "wallTime":{"$date":...},"ns":{"db":"sample_restaurants","coll":"restaurants"},
+   "documentKey":{"_id":{"$oid":"..."}},"updateDescription":{"updatedFields":
+   {"cuisine":"Irish"},"removedFields":[],"truncatedArrays":[]}}}
+
+Modify the Change Stream Output
+-------------------------------
+
+To modify the change stream output, you can pass pipeline stages in an array as a
+parameter to the ``watch()`` method. You can include the following stages in the
+array:
+
+- ``$addFields`` or ``$set``: Adds new fields to documents
+- ``$match``: Filters the documents
+- ``$project``: Projects a subset of the document fields
+- ``$replaceWith`` or ``$replaceRoot``: Replaces the input document with the
+  specified document
+- ``$redact``: Restricts the contents of the documents
+- ``$unset``: Removes fields from documents
+
+The following passes a pipeline that includes the ``$match`` stage to the ``watch()``
+method. This instructs the ``watch()`` method to output only update operations:
+
+.. literalinclude:: /includes/read/change-streams.php
+   :start-after: start-change-stream-pipeline
+   :end-before: end-change-stream-pipeline
+   :language: php
+   :dedent:
+
+Modify ``watch()`` Behavior
+---------------------------
+
+To modify the behavior of the ``watch()`` method, you can pass an options array
+as a parameter to ``watch()``. The following table describes some options you
+can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``fullDocument``
+     - | Specifies whether to show the full document after the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`php-change-stream-pre-post-image`.
+
+   * - ``fullDocumentBeforeChange``
+     - | Specifies whether to show the full document as it was before the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`php-change-stream-pre-post-image`.
+    
+   * - ``resumeAfter``
+     - | Instructs ``watch()`` to resume returning changes after the
+         operation specified in the resume token.
+       | Each change stream event document includes a resume token as the ``_id``
+         field. Pass the entire ``_id`` field of the change event document that
+         represents the operation you want to resume after.
+       | This option is mutually exclusive with ``startAfter`` and ``startAtOperationTime``.
```

Comment:
S: since you mention `startAtOperationTime`, maybe you can include it in this table